File: tests/fixtures/lua/basic.lua
local json = require("json")
http = require "socket.http"

-- module comment
--[[
block comment
]]

local VERSION = "1.0"
Config = { enabled = true }

local function helper(value)
  return value
end

function greet(name)
  return "hi " .. name
end

User = {}
function User:new(id)
  return { id = id }
end

run = function(opts)
  return opts
end

File: tests/fixtures/rust/basic.rs
use std::{fmt, io};
pub use crate::prelude::*;
mod inner;
extern crate alloc;

// module comment
/* block
comment */

pub const VERSION: &str = "1.0";
static COUNT: usize = 3;

pub trait Named {
    fn name(&self) -> &str;
}

pub struct User {
    pub id: u64,
}

enum State {
    Ready,
}

type UserId = u64;

pub async fn load(id: UserId) -> Result<User, Error> {
    todo!()
}

fn helper(value: &str) -> String {
    value.to_owned()
}
